### PR TITLE
[zh]Sync content/zh/docs/tasks/traffic-management/egress/egress-tls-o…

### PR DESCRIPTION
**Pull request by yanrongshi**
```yaml
content/zh/docs/ops/best-practices/image-signing-validation/index.md
```

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Cosign 是作为 [sigstore](https://www.sigstore.dev) 项目的一部分开发的工具。
+它简化了已签名的开放容器倡议（OCI）制品（例如容器镜像）的签名和验证。
+
+从 Istio 1.12 开始，我们签署所有正式发布的容器镜像作为我们发布过程的一部分。
+然后，最终用户可以使用下面描述的过程来验证这些镜像。
+
+此过程适用于手动执行或与构建或部署管道集成以自动验证镜像制品。
+
+
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/image-signing-validation/index.md:21]: Fix one error:
```suggestion

```

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+1. 验证 `cosign` 二进制签名：
+
+   {{< text bash >}}
+$ openssl dgst -sha256 \
+    -verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pub) \
+    -signature <(cat /path/to/cosign.sig | base64 -d) \
+    /path/to/cosign-binary
+    {{< /text >}}
+
+1. 使二进制文件可执行（'chmod + x'），并移动到 'PATH' 上的一个位置。
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/image-signing-validation/index.md:36]: Need to run this locally to see the highlighting. Not sure if it's doing something with the comma or is it a "`" va "'" on the line.

Comment (yanrongshi) [reply content/zh/docs/ops/best-practices/image-signing-validation/index.md:35]: I have try to `make lint`,However it appears as follows


image

Comment (ericvn) [reply content/zh/docs/ops/best-practices/image-signing-validation/index.md]: You need to wait longer for the actual lint messages. Sample output for another PR.
Screen Shot 2022-09-07 at 11 41 47 AM

Comment (yanrongshi) [reply content/zh/docs/ops/best-practices/image-signing-validation/index.md]: image

**Comment by ericvn** [comment]
Screen Shot 2022-09-07 at 11 56 23 AM

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+1. 验证 `cosign` 二进制签名：
+
+   {{< text bash >}}
+$ openssl dgst -sha256 \
+    -verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pub) \
+    -signature <(cat /path/to/cosign.sig | base64 -d) \
+    /path/to/cosign-binary
+    {{< /text >}}
+
+1. 使二进制文件可执行（chmod + x），并移动到 'PATH' 上的一个位置。
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/image-signing-validation/index.md:35]:
```suggestion
1. 使二进制文件可执行（`chmod + x`），并移动到 'PATH' 上的一个位置。
```

Comment (yanrongshi) [reply content/zh/docs/ops/best-practices/image-signing-validation/index.md:35]: That was true before, but no, I'm trying again locally

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+1. 验证 `cosign` 二进制签名：
+
+   {{< text bash >}}
+$ openssl dgst -sha256 \
+    -verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pub) \
+    -signature <(cat /path/to/cosign.sig | base64 -d) \
+    /path/to/cosign-binary
+    {{< /text >}}
+
+1. 使二进制文件可执行（`chmod + x`），并移动到 'PATH' 上的一个位置。
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/image-signing-validation/index.md:35]: You don't want a space between the + and the x, and backpacking PATH:
```suggestion
1. 使二进制文件可执行（`chmod +x`），并移动到 `PATH` 上的一个位置。
```

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md (hunk 3 above)
Comment (yanrongshi) [reply content/zh/docs/ops/best-practices/image-signing-validation/index.md]: > 

Thank you very much for your careful help. I feel ashamed for my lack of rigorous document working spirit

**Comment by ericvn** [comment]
That should get the lint passing and the various red highlighted words similar to the English page. I'll leave the rest to the Chinese approvers.

**Comment by yanrongshi** [comment]
> That should get the lint passing and the various red highlighted words similar to the English page. I'll leave the rest to the Chinese approvers.

Well, thank you very much for your review

File: content/zh/docs/ops/best-practices/image-signing-validation/index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: 图像签名和验证
```
Comment (kebe7jun) [inline content/zh/docs/ops/best-practices/image-signing-validation/index.md:2]: 镜像。

**Review by windsonsea — APPROVED** [review]

**Pull request merged**